stdlib/3/tkinter/__init__.pyi

```diff
@@ -688,10 +688,10 @@ class Place:
         height: _ScreenUnits = ...,
         x: _ScreenUnits = ...,
         y: _ScreenUnits = ...,
-        relheight: float = ...,
-        relwidth: float = ...,
-        relx: float = ...,
-        rely: float = ...,
+        relheight: _ScreenUnits = ...,
+        relwidth: _ScreenUnits = ...,
+        relx: _ScreenUnits = ...,
+        rely: _ScreenUnits = ...,
```

Comment:
I think `float` is correct for these:

```
>>> import tkinter
>>> l=tkinter.Label()
>>> l.place(relx='2c')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.7/tkinter/__init__.py", line 2188, in place_configure
    + self._options(cnf, kw))
_tkinter.TclError: expected floating-point number but got "2c"
>>> 
```

Do all `_ScreenUnits` work for you? I wouldn't expect it to work because the `rel` in these is short for "relative", so 0.2 means 20% and so on. So what should `3px` mean as a relative value?

Reply:
You are right! I must check it better, no time just now.  
tcl manual:
-relx
     In this case the location is specified in a relative fashion as a floating-point number: 0.0 corresponds to the left edge of the master and 1.0 corresponds to the right edge of the master. Location need not be in the range 0.0-1.0. If both -x and -relx are specified for a slave then their values are summed. For example, -relx 0.5 -x -2 positions the left edge of the slave 2 pixels to the left of the center of its master.

-rely location
    In this case the value is specified in a relative fashion as a floating-point number: 0.0 corresponds to the top edge of the master and 1.0 corresponds to the bottom edge of the master. Location need not be in the range 0.0-1.0. If both -y and -rely are specified for a slave then their values are summed. For example, -rely 0.5 -x 3 positions the top edge of the slave 3 pixels below the center of its master.

Reply:
The type should be Union[str, float], the same of _ScreenUnits but without the name.

Reply:
Why should strings be accepted?

Reply:
Because i want to do:
```
w1 = tk.Label()
w1.place(relx=0.5)
w2 = tk.Label()
w2.place(**w1.place_info())
```
without this error:
`error: Argument "relx" has incompatible type "str"; expected "float"`